sdks/python/apache_beam/io/tfrecordio.py

```diff
@@ -0,0 +1,274 @@
+               coder=coders.BytesCoder(),
+               compression_type=fileio.CompressionTypes.AUTO,
+               **kwargs):
+    """Initialize a ReadFromTFRecord transform.
+
+    Args:
+      file_pattern: The file pattern to read from as a local file path or a GCS
+        gs:// path. The path can contain glob characters (*, ?, and [...]
+        sets).
+      coder: Coder used to decode each record.
+      compression_type: Used to handle compressed input files. Typical value
+          is CompressionTypes.AUTO, in which case the file_path's extension will
+          be used to detect the compression.
+      **kwargs: optional args dictionary. These are passed through to parent
+        constructor.
+
+    Returns:
+      A WriteToTFRecord transform object.
+    """
+    super(ReadFromTFRecord, self).__init__(**kwargs)
+    self._args = (file_pattern, coder, compression_type)
+
+  def expand(self, pvalue):
+    return pvalue.pipeline | Read(_TFRecordSource(*self._args))
+
+
+class _TFRecordSink(fileio.FileSink):
+  """Sink for writing TFRecords files.
+
+  For detailed TFRecord format description see:
+    https://www.tensorflow.org/versions/master/api_docs/python/python_io.html#tfrecords-format-details
+  """
+
+  def __init__(self, file_path_prefix, coder, file_name_suffix, num_shards,
+               shard_name_template, compression_type):
+    """Initialize a TFRecordSink. See WriteToTFRecord for details."""
+
+    super(_TFRecordSink, self).__init__(
+        file_path_prefix=file_path_prefix,
+        coder=coder,
+        file_name_suffix=file_name_suffix,
+        num_shards=num_shards,
+        shard_name_template=shard_name_template,
+        mime_type='application/octet-stream',
+        compression_type=compression_type)
+
+  def write_encoded_record(self, file_handle, value):
+    _TFRecordUtil.write_record(file_handle, value)
+
+
+class WriteToTFRecord(PTransform):
+  """Transform for writing to TFRecord sinks."""
+
+  def __init__(self,
+               file_path_prefix,
+               coder=coders.BytesCoder(),
+               file_name_suffix='',
+               num_shards=0,
+               shard_name_template=fileio.DEFAULT_SHARD_NAME_TEMPLATE,
+               compression_type=fileio.CompressionTypes.AUTO,
+               **kwargs):
+    """Initialize WriteToTFRecord transform.
+
+    Args:
+
+      file_path_prefix: The file path to write to. The files written will begin
+        with this prefix, followed by a shard identifier (see num_shards), and
+        end in a common extension, if given by file_name_suffix. In most cases,
+        only this argument is specified and num_shards, shard_name_template, and
+        file_name_suffix use default values.
+      coder: Coder used to encode each record.
+      file_name_suffix: Suffix for the files written.
+      num_shards: The number of files (shards) used for output. If not set, the
+        service will decide on the optimal number of shards.
```

Comment:
Remove the reference to the service. I think the below should be enough:

`The number of files (shards) used for output. If not set the default value will be used.`